config/software/python.rb

```diff
@@ -33,13 +33,21 @@
   "LDFLAGS" => "-Wl,-rpath,#{install_dir}/embedded/lib -L#{install_dir}/embedded/lib"
 }
 
+python_configure = ["./configure",
+                    "--enable-universalsdk=/",
+                    "--prefix=#{install_dir}/embedded"]
+
+if ENV['PKG_TYPE'] == 'dmg'
+  python_configure.push('--enable-ipv6', '--with-universal-archs=intel')
+end
+
+python_configure.push("--with-dbmliborder=")
```

Comment:
It needs to be the last argument, otherwise it fails. Not really beautiful, I'm open to suggestions!